### PR TITLE
io complete

### PR DESCRIPTION
**Pull request by Victoriam7**
I/O functionality complete for now

### CODE REVIEW

**Review by jpakey99 — DISMISSED** [review]

File: backend/src/pipelines.py [hunk 1]
```diff
@@ -1,29 +1,75 @@
...
-    processing_pipeline(ifiles)
+    processing_pipeline(files)
 
 Authors:
     Brendan Grau <bsg8376@rit.edu>
 
 License:
     © 2022 BeyondRGB
-    This code is licensed under the MIT license (see LICENSE.txt for details)
+    This code is licensed under the MIT license (see LINSE.txt for details)
```
Comment (egc2846) [inline backend/src/pipelines.py:12]: typo

**Review by egc2846 — CHANGES_REQUESTED** [review]

File: backend/src/pipelines.py [hunk 2]
```diff
@@ -1,29 +1,75 @@
 """ pipelines.py
-Collection of all pipelines
+llection of all pipelines
```
Comment (sk4900) [inline backend/src/pipelines.py:2]: typo

**Review by sk4900 — APPROVED** [review]

**Review by egc2846 — APPROVED** [review]

File: backend/src/pipelines.py (hunk 1 above)
Comment (Victoriam7) [reply backend/src/pipelines.py]: Fixed

File: backend/src/pipelines.py (hunk 2 above)
Comment (Victoriam7) [reply backend/src/pipelines.py]: fixed

**Review by sk4900 — APPROVED** [review]

**Review by keenanm500 — APPROVED** [review]

**Pull request merged**